src/main/java/frc/robot/commands/test/SteerFeedForwardCharacterzationSingle.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.test;
+
+
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Module;
+import frc.robot.util.FeedForwardCharacterizationData;
+
+
+/** Add your docs here. */
+public class SteerFeedForwardCharacterzationSingle extends CommandBase {
+  double value = 0;
+  FeedForwardCharacterizationData m_feedForwardCharacterizationData;
+  Module m_module;
+
+  Timer m_timer = new Timer();
+  Drivetrain m_drive;
+  SendableChooser<Module> m_moduleChooser;
+
+  public SteerFeedForwardCharacterzationSingle(Drivetrain drive, SendableChooser<Module> moduleChooser) {
+    m_drive = drive;
+    m_moduleChooser = moduleChooser;
+    addRequirements(drive);
+  }
+
+  public void initialize() {
+    m_timer.start();
+    m_drive.setAllOptimize(false);
+    m_feedForwardCharacterizationData = new FeedForwardCharacterizationData();
+    this.m_module = m_moduleChooser.getSelected();
+  }
+
+  public void execute() {
+    runCharacterizationVolts();
+    if (m_timer.get() > 0.5) {
+      m_feedForwardCharacterizationData.add(m_module.getSteerVelocity(), value);
```

Comment:
Can we characterize all 4 motors simultaneously by keeping  a 4 element array of FeedForwardCharacterizationData and recording the data in appropriate places? Then, there is no need to have a module chooser, and all procedure will go faster.

Reply:
Right now we decided that running them simultaneously causes too much noise to characterize each one properly and running each one right after another exists but isn't in PR as it sometimes randomly spits a out of index error